packages/monitors/index.js

```diff
@@ -72,21 +74,51 @@ async function run({
       endingBlock,
       monitorConfig,
       tokenPriceFeedConfig,
-      medianizerPriceFeedConfig
+      medianizerPriceFeedConfig,
+      denominatorPriceFeedConfig
     });
 
     const getTime = () => Math.round(new Date().getTime() / 1000);
 
     const networker = new Networker(logger);
 
+    // We want to enforce that all pricefeeds return prices in the same precision, so we'll construct one price feed
+    // initially and grab its precision to pass into the other price feeds:
+    const medianizerPriceFeed = await createReferencePriceFeedForEmp(
+      logger,
+      web3,
+      networker,
+      getTime,
+      empAddress,
+      medianizerPriceFeedConfig
+    );
+    const priceFeedDecimals = medianizerPriceFeed.getPriceFeedDecimals();
+
     // 0. Setup EMP and token instances to monitor.
-    const [networkId, latestBlock, tokenPriceFeed, medianizerPriceFeed] = await Promise.all([
+    const [networkId, latestBlock, tokenPriceFeed, denominatorPriceFeed] = await Promise.all([
       web3.eth.net.getId(),
       web3.eth.getBlock("latest"),
-      createTokenPriceFeedForEmp(logger, web3, networker, getTime, empAddress, tokenPriceFeedConfig),
-      createReferencePriceFeedForEmp(logger, web3, networker, getTime, empAddress, medianizerPriceFeedConfig)
+      createTokenPriceFeedForEmp(logger, web3, networker, getTime, empAddress, {
```

Comment:
Here's some new logic: I enforce that all of the pricefeeds use the same precision.